src/main/kotlin/io/zerobase/smarttracing/notifications/AmazonEmailSender.kt

```diff
@@ -22,7 +23,7 @@ fun RawEmailRequestBuilder.bytes(bytes: ByteArray): RawEmailRequestBuilder {
     }
 }
 
-class AmazonEmailSender(
+class AmazonEmailSender @Inject constructor(
```

Comment:
not super familiar with the side effects of using guice / dependency injections; why is this annotation required here?

Reply:
dependency injection requires that you either have a no-argument constructor or a constructor that is explicitly marked as injectable. the annotation is how you mark it as injectable.